lms/static/sass/elements/_controls.scss

```diff
@@ -403,3 +403,22 @@
       margin-bottom: none;
     }
   }
+
+  .btn-link {
```

Comment:
@marcotuts I've addressed your comment about the link button. Somehow i've to add `!important` for focus & hover state, i don't know why. Please have a look and let me know if i'm missing something. Thanks a lot.